### PR TITLE
[ui] Token Stats

### PR DESCRIPTION
**Pull request by iverc**

### CODE REVIEW

File: ui/app/src/components/shared/Header.vue [hunk 1]
```diff
@@ -1,37 +1,37 @@
...
     <div class="secondary-header-container">Welcome to our BetaNet! Please be aware that while this has passed several public security audits, it is still in Beta. We encourage you to first understand how it works before transacting on it and exercise caution at all times. <a target="_blank" href="https://docs.sifchain.finance/resources/betanet-launch">Here's a list of security measures taken</a>, but please note that all transactions conducted are at your own risk.</div>
     <div class="primary-header-container">
       <div>
         <Menu />
-        <!-- <SifSwitch label="Typeface Cursive" v-model="typefaceCursive" /> -->
       </div>
-      <div>
-        <slot name="right"></slot>
+      <div class="third-container">
+        <Suspense>
```
Comment (59023g) [inline ui/app/src/components/shared/Header.vue:22]: What's `Suspense` ?

**Review by 59023g — APPROVED** [review]
Looks good. And lightweight. Nice. Only request would be to have more clear way to go back to trading view, from stats page.

**Review by 59023g — APPROVED** [review]

File: ui/app/src/components/shared/Header.vue (hunk 1 above)
Comment (iverc) [reply ui/app/src/components/shared/Header.vue:22]: @59023g It makes possible to use `async start()`, and does not show component until start is done loading.

**Review by thomasdavis — APPROVED** [review]
LGTM. 

I'm not if those config changes are relevant or not though. I don't know enough.

**Pull request merged**